pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/AdvertisedListenersTest.java

```diff
@@ -112,6 +112,7 @@ public void testLookup() throws Exception {
         // Verify we can get the correct HTTP redirect to the advertised listener
         for (PulsarAdmin a : getAllAdmins()) {
             TopicStats s = a.topics().getStats("my-topic");
+            a.lookups().lookupTopic("my-topic");
```

Comment:
What's the purpose of this line? The returned value of `lookupTopic` is not validated.

Reply:
To make sure the lookup operation will not get exceptions, I have added a notNull check for the returned service URL.